src/Microsoft.Health.Fhir.Liquid.Converter/Processors/JsonToHl7v2Processor.cs

```diff
@@ -56,7 +69,7 @@ public string Convert(JObject data, string rootTemplate, ITemplateProvider templ
         {
             var jsonData = data.ToObject();
             var result = InternalConvertFromObject(jsonData, rootTemplate, templateProvider, traceInfo);
-            var hl7Message = GenerateHL7Message(JObject.Parse(result));
+            var hl7Message = GenerateHL7Message(ConvertToJObject(result));
```

Comment:
@pallar-ms @dustinburson This class also uses the updated `JsonProcessor` which defaults to treating dates as `strings`. Because of this I update this `post processing` step to also treat dates as strings.

As @pallar-ms  mentioned there are a few other places where `JObject.Parse` is used. We can see if they need to be updated in as separate PR.

- [Validate.cs](https://github.com/microsoft/FHIR-Converter/blob/main/src/Microsoft.Health.Fhir.Liquid.Converter/DotLiquids/Validate.cs#L74)
- [TemplateUtility.cs](https://github.com/microsoft/FHIR-Converter/blob/main/src/Microsoft.Health.Fhir.Liquid.Converter/Utilities/TemplateUtility.cs#L162)

Reply:
Great! That makes things easier